setup.py

```diff
@@ -9,45 +9,58 @@
 """
 
 # Standard Python Libraries
+import codecs
 from glob import glob
-import io
-from os.path import basename, splitext
+from os.path import abspath, basename, dirname, join, splitext
 
 # Third-Party Libraries
 from setuptools import find_packages, setup
 
 
 def readme():
     """Read in and return the contents of the project's README.md file."""
-    # Python 3
-    try:
-        with open("README.md", encoding="utf-8") as f:
-            return f.read()
-    # Python 2 fallback
-    except TypeError:
-        with io.open("README.md", encoding="utf-8") as f:
-            return f.read()
+    with open("README.md", encoding="utf-8") as f:
+        return f.read()
 
 
-def package_vars(version_file):
-    """Read in and return the variables defined by the version_file."""
-    pkg_vars = {}
-    with open(version_file) as f:
-        exec(f.read(), pkg_vars)  # nosec
-    return pkg_vars
+# Below two methods were pulled from:
+# https://packaging.python.org/guides/single-sourcing-package-version/
+def read(rel_path):
+    """Open a file for reading from a given relative path."""
+    here = abspath(dirname(__file__))
+    with codecs.open(join(here, rel_path), "r") as fp:
+        return fp.read()
+
+
+def get_version(version_file):
+    """Extract a version number from the given file path."""
+    for line in read(version_file).splitlines():
+        if line.startswith("__version__"):
+            delim = '"' if '"' in line else "'"
+            return line.split(delim)[1]
+    raise RuntimeError("Unable to find version string.")
 
 
 setup(
     name="cyhy-feeds",
-    version=package_vars("aws_jobs/_version.py")["__version__"],
-    author="Cyber and Infrastructure Security Agency",
-    author_email="ncats@hq.dhs.gov",
-    url="https://www.us-cert.gov/resources/ncats",
-    download_url="https://github.com/cisagov/cyhy-feeds",
-    license="License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+    # Versions should comply with PEP440
+    version=get_version("aws_jobs/_version.py"),
     description="Data feed components for Cyber Hygiene",
     long_description=readme(),
     long_description_content_type="text/markdown",
+    # Landing page for CISA's cybersecurity mission
+    url="https://www.cisa.gov/cybersecurity",
+    # Additional URLs for this project per
+    # https://packaging.python.org/guides/distributing-packages-using-setuptools/#project-urls
+    project_urls={
+        "Source": "https://github.com/cisagov/skeleton-python-library",
+        "Tracker": "https://github.com/cisagov/skeleton-python-library/issues",
```

Comment:
These URLs should be updated.

Reply:
Since @jsf9k is somehow not an owner in this repo, but he should get credit for this:
```suggestion
        "Source": "https://github.com/cisagov/cyhy-feeds",
        "Tracker": "https://github.com/cisagov/cyhy-feeds/issues",
```

Reply:
My buffoonery knows no bounds.  :laughing: